sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/postgres/PostgresInt2Tests.java

```diff
@@ -0,0 +1,397 @@
+                        "\n" +
+                        "SELECT i.f1, i.f1 + '2'::INT2 AS x FROM INT2_TBL i WHERE f1 < 32766;\n" +
+                        "   f1   |   x    \n" +
+                        "--------+--------\n" +
+                        "      0 |      2\n" +
+                        "   1234 |   1236\n" +
+                        "  -1234 |  -1232\n" +
+                        " -32767 | -32765\n" +
+                        "(4 rows)\n" +
+                        "\n" +
+                        "SELECT i.f1, i.f1 + '2'::INT4 AS x FROM INT2_TBL i;\n" +
+                        "   f1   |   x    \n" +
+                        "--------+--------\n" +
+                        "      0 |      2\n" +
+                        "   1234 |   1236\n" +
+                        "  -1234 |  -1232\n" +
+                        "  32767 |  32769\n" +
+                        " -32767 | -32765\n" +
+                        "(5 rows)\n" +
+                        "\n" +
+                        "SELECT i.f1, i.f1 - '2'::INT2 AS x FROM INT2_TBL i WHERE f1 > -32767;\n" +
+                        "  f1   |   x   \n" +
+                        "-------+-------\n" +
+                        "     0 |    -2\n" +
+                        "  1234 |  1232\n" +
+                        " -1234 | -1236\n" +
+                        " 32767 | 32765\n" +
+                        "(4 rows)\n" +
+                        "\n" +
+                        "SELECT i.f1, i.f1 - '2'::INT4 AS x FROM INT2_TBL i;\n" +
+                        "   f1   |   x    \n" +
+                        "--------+--------\n" +
+                        "      0 |     -2\n" +
+                        "   1234 |   1232\n" +
+                        "  -1234 |  -1236\n" +
+                        "  32767 |  32765\n" +
+                        " -32767 | -32769\n" +
+                        "(5 rows)\n" +
+                        "\n" +
+                        "SELECT i.f1, i.f1 / '2'::INT4 AS x FROM INT2_TBL i;\n" +
+                        "   f1   |   x    \n" +
+                        "--------+--------\n" +
+                        "      0 |      0\n" +
+                        "   1234 |    617\n" +
+                        "  -1234 |   -617\n" +
+                        "  32767 |  16383\n" +
+                        " -32767 | -16383\n" +
+                        "(5 rows)\n" +
+                        "\n" +
+                        "SELECT i.f1, i.f1 / '2'::INT4 AS x FROM INT2_TBL i;\n" +
+                        "   f1   |   x    \n" +
+                        "--------+--------\n" +
+                        "      0 |      0\n" +
+                        "   1234 |    617\n" +
+                        "  -1234 |   -617\n" +
+                        "  32767 |  16383\n" +
+                        " -32767 | -16383\n" +
+                        "(5 rows)\n" +
+                        "\n"
+        );
+    }
+
+    @Test
+    public void testBadSelect() {
```

Comment:
this is not really a test of integers, maybe we should put it somewhere else.
I already have some negative parser tests, but I am not sure that's the right place.

Reply:
Okay, I put there because it was taken from `int2.out`. It also relies on the `prepareData`. 

We could remove it if it doesn't belong here.

Reply:
If it relies on the preparedData let's keep it. If we keep the same file contents as the Postgres test files it's easier to see what is done and what not.